Gemfile.lock

```diff
@@ -54,7 +54,7 @@ GEM
     jquery-rails (3.1.2)
       railties (>= 3.0, < 5.0)
       thor (>= 0.14, < 2.0)
-    json (1.8.1)
+    json (1.8.3)
```

Comment:
Hey, I noticed this is just a bugfix version, but why do we need to update this?